services/static-webserver/client/source/class/osparc/dashboard/CardBase.js

```diff
@@ -299,23 +297,23 @@ qx.Class.define("osparc.dashboard.CardBase", {
 
     __applyResourceData: function(resourceData) {
       let uuid = null;
-      let owner = "";
-      let defaultHits = null;
+      let owner = null;
       let workbench = null;
+      let defaultHits = null;
       switch (resourceData["resourceType"]) {
         case "study":
-          uuid = resourceData.uuid ? resourceData.uuid : uuid;
-          owner = resourceData.prjOwner ? resourceData.prjOwner : owner;
-          workbench = resourceData.workbench ? resourceData.workbench : workbench;
+          uuid = resourceData.uuid ? resourceData.uuid : null;
+          owner = resourceData.prjOwner ? resourceData.prjOwner : "";
+          workbench = resourceData.workbench ? resourceData.workbench : {};
           break;
         case "template":
-          uuid = resourceData.uuid ? resourceData.uuid : uuid;
-          owner = resourceData.prjOwner ? resourceData.prjOwner : owner;
-          workbench = resourceData.workbench ? resourceData.workbench : workbench;
+          uuid = resourceData.uuid ? resourceData.uuid : null;
+          owner = resourceData.prjOwner ? resourceData.prjOwner : "";
```

Comment:
TIP: i would have a mechanism that loads all these defaults from the models in services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

Reply:
👌

Reply:
hmm, can I get in the loop please? I am not sure I get the point of having delete rights on the services.